src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java

```diff
@@ -200,7 +207,7 @@ public void clickBottoneConfermaDelega(){
 
     public boolean verificaEsistenzaErroreCodiceSbagliato() {
         try {
-            By esistenzaBy = By.id("alert-api-status");
+            By esistenzaBy = By.id("alert-api-status}");
```

Comment:
I guess the `}` shouldn't be there

Reply:
because in the front end the is that

Reply:
I spoke with @alessandrogelmi, and he fixed it, so we might wait for the next FE releases.